client/src/components/friends-page/FriendsContent.css

```diff
@@ -16,7 +15,7 @@
 }
 
 .clouds-wrapper {
-  height: 100%;
+  height: 90vh;
   display: flex;
   flex-direction: column;
   width: 100vw;
```

Comment:
Just the Monk inside me. You might wanna stick to a order within your css rulsesets.
like
    From outside to insde
    Layout Properties (position, float, clear, display)
    Box Model Properties (width, height, margin, padding)
    Visual Properties (color, background, border, box-shadow)
    Typography Properties (font-size, font-family, text-align, text-transform)
    Misc Properties (cursor, overflow, z-index)

Reply:
i like this strategy! will consider...